ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/compliance.coverage.utils.tsx

```diff
@@ -116,12 +116,12 @@ const statusIconTextMap: { [key in ComplianceCheckStatus]: ClusterStatusObject }
     [ComplianceCheckStatusEnum.PASS]: {
         icon: (
             <Icon>
-                <CheckCircleIcon color="var(--pf-v5-global--success-color--100)" />
+                <CheckCircleIcon color="var(--pf-v5-global--primary-color--100)" />
```

Comment:
I wonder why UX decided to use `blue` instead of `green` for a `Pass` 🤔

Reply:
Not sure tbh, I never explicitly asked. Seems like a good question for @alanonthegit